maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt

```diff
@@ -2607,6 +2607,7 @@ class IntegrationTest {
                 "argumentA",
                 "argumentB",
                 "argumentC",
+                "argumentA=argumentAValue"
```

Comment:
Using = is a bit strange because it's not common in YAML. It feels to me that `:` is better, but kinda breaks when booleans exists
```yaml
- launchApp:
     arguments:
         - argumentA: valueA 
         - argumentB
         - argumentC: valueC
```

So my suggestion would be something like this:
```yaml
- launchApp:
     arguments:
         - argumentA: valueA 
         - argumentB: True
         - argumentC: valueC
```

What do you think?

Reply:
Toatally, agreed I'll add a `:` instead.

Reply:
Done ✅